### ISSUE TITLE
Intermittently missing "All classes" back link on coach pages

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#7381]
### Observed behavior

There's no link back to "All classes" although I'm pretty sure "Hack session school" has multiple classes

![image](https://user-images.githubusercontent.com/10248067/88345085-0e2de300-ccfa-11ea-9168-95851aa147ca.png)

Other details:

* This class was initially imported with a facility from a different server
* The full class list is still accessible, but you have to find it

### Expected behavior

The link is there (screenshot from kolibri demo)

![image](https://user-images.githubusercontent.com/10248067/88345252-71b81080-ccfa-11ea-97bb-6a51d1e625ed.png)

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

**Comment by nucleogenesis** [comment]
@jonboiser Do you see any console errors there?

I'm not seeing this on 0.14 with the same imported facility on any of the classes.

**Comment by jonboiser** [comment]
No console errors, but I did confirm that it's kind of random. 

It's not there 100% of the time if I refresh on the Home Page.

It's not there if I go directly from All Classes > Jessica's Class on the first load, but is there a little less than 100% (90%?) of the time if I go back and forth between those two pages.

So my hunch is that it must be some kind of race condition in loading the total number of classes. I'm assuming that's what determines whether or not the back link appears.

**Comment by nucleogenesis** [comment]
@jonboiser I've tried a bunch of different ways to get the link to stop appearing but I cannot. Perhaps I'm not understanding your replication instructions?

I've refreshed, went back and forth between pages, loaded straight to All Classes, Reports, Plan - then refreshed and/or navigated back to All Classes never seeing the reported error in Chrome or Firefox. I also slowed my network down in dev tools without any issues :-/
